baseline.yaml

```diff
@@ -404,16 +404,17 @@ criteria:
     scorecard_probe:
       - # None, may not be suitable
 
-  - id: OSPS-DO-03
+  - id: OSPS-SA-01
     maturity_level: 2
-    category: Documentation
+    category: Threat Modeling
```

Comment:
Need to either update the "category" label here, or auto-generate it from the `id`.